07.bitstream.semantics.md

```diff
@@ -1314,6 +1315,41 @@ coded video sequence.
 number of bits used to represent time_offset_value is equal to time_offset_length. When
 time_offset_value is not present, its value is inferred to be equal to 0.
 
+#### Metadata crop semantics
+
+When present the metadata_crop OBU applies starting at the next frame in the sequence 
+with matching temporal_id and spatial_id, and shall apply to all matching frames until the next Key Frame
```

Comment:
> When present the metadata_crop OBU applies starting at the next frame in the sequence
with matching temporal_id and spatial_id, ...

Alexis and myself are wondering here what the behavior should be if no extension header is present. Perhaps in that case it should apply to all layers?

Also instead of:
> and shall apply to all matching frames

it would be better to write "and shall apply to all frames with a matching temporal_id and spatial_id"

Reply:
Yes.  I think it should apply to all layers if no header is present.  I will add wording clarifying that.